engine/class_modules/apl/apl_death_knight.cpp

```diff
@@ -108,32 +108,33 @@ void blood( player_t* p )
   default_->add_action( "raise_dead" );
   default_->add_action( "blooddrinker,if=!buff.dancing_rune_weapon.up&(!covenant.night_fae|buff.deaths_due.remains>7)" );
   default_->add_action( "blood_boil,if=charges>=2&(covenant.kyrian|buff.dancing_rune_weapon.up)" );
-  default_->add_action( "raise_dead" );
+//  default_->add_action( "raise_dead" );
```

Comment:
Why remove raise dead here? its not on a GCD, and is just a free dps increase? Doesn't make much sense to remove it.

Reply:
Cause it's used 2 lines before, and you want to use it before blooddrinker so you don't interrupt blooddrinker.